src/app/tabs/camera.cpp

```diff
@@ -22,9 +25,20 @@ CameraTab::CameraTab(QWidget *parent) : QWidget(parent)
     layout->addWidget(this->local_camera_widget());
     layout->addWidget(this->network_camera_widget());
 
-    connect(this, &CameraTab::disconnected, [layout]() { layout->setCurrentIndex(0); });
+    connect(this, &CameraTab::disconnected, [layout,this]() {
```

Comment:
i dont see this signal ever being emitted

Reply:
Line 313 of this file

Reply:
ah it was hidden by the diff lol